format/Message.fbs

```diff
@@ -62,10 +167,41 @@ table RecordBatch {
   buffers: [Buffer];
 }
 
+/// A data header describing the shared memory layout of a "record" or "row"
+/// batches that supports sparsity and compressoin.
+/// Some systems call this a "sparse row batch" internally and others a
+/// "sparserecord batch".
+table SparseRecordBatch {
+  /// number of records / rows. The arrays in the batch should all have this
+  /// length
+  length: long;
+
+  /// Nodes correspond to the pre-ordered flattened logical schema.  Not all
+  /// fields are necessarily present.  Specifically, "empty" fields may be
```

Comment:
What does "not present" mean? It's not in the list? It's null or something?

Reply:
Not in the list, I will clarify.